recipe/scripts/activate-binutils.sh

```diff
@@ -1,5 +1,8 @@
 # shellcheck shell=sh
 
+# This script expands variables that potentially do not exist. In case -u is enabled, this will fail.
+set +u
```

Comment:
wouldn't it be better to fix the problematic calls in this scripts (i.e. add a fall-back or error if they're not defined), rather than just papering over this issue?

Aside from the problem that this would persist `+u` even if before it had been `-u` before the activation (which is the kind of leak that we definitely want to avoid)